src/components/Comment/style.scss

```diff
@@ -1,100 +1,204 @@
+            width: 90px;
+            padding-top: 8px;
+            cursor: pointer;
+        }
 
-            color: #333;
+        input[type='checkbox'] {
+            display: none;
+            visibility: hidden;
         }
-    }
 
-    &__replies {
-        list-style-type: none;
-        padding: 0;
+        label::before {
+            content: '';
+            display: inline-block;
+            height: 26px;
+            width: 26px;
+            margin-left: 5px;
+            background-color: #fff;
+            border: 1px solid #fff;
+            border-radius: 3px;
+            position: absolute;
+            top: 3px;
+            right: 0px;
+        }
+
+        label::after {
+            content: '';
+            color: #000;
+            display: inline-block;
+            height: 6px;
+            width: 16px;
+            border-left: 2px solid;
+            border-bottom: 2px solid;
+            transform: rotate(-45deg);
+            position: absolute;
+            right: 4px;
+            top: 10px;
+        }
+
+        input[type='checkbox'] + label::after {
+            visibility: hidden;
+        }
+
+        input[type='checkbox']:checked + label::after {
+            visibility: visible;
+        }
+
+        input[type='checkbox']:focus + label::before {
+            outline: rgb(59, 153, 252) auto 5px;
+        }
     }
 
-    &__edit-actions,
-    &__reply-actions {
-        padding-right: 10px;
-        padding-bottom: 10px;
-        width: 100%;
-        background-color: #f7f7f7;
-        text-align: right;
-        box-sizing: border-box;
+    &__confirm-delete,
+    &__error {
+        color: #fff;
+        font-weight: bold;
+        font-size: 0.8em;
 
         button {
-            margin-right: 5px;
+            margin-left: 5px;
+            margin-right: 0px;
         }
     }
 
-    &__confirm-delete {
-        margin-right: 5px;
+    &__error {
+        background-color: #b4191f;
+        border-radius: 3px;
+        padding: 5px;
+
+        button {
+            float: right;
+            margin-left: 5px;
+            background-color: #901419;
+            border-color: #901419;
+            padding: 2px;
+            padding-left: 10px;
+            padding-right: 10px;
+            font-size: 0.9em;
+            font-weight: bold;
+        }
     }
 
-    &__error {
-        color: red;
+    &__progress {
+        color: #fff;
         font-weight: bold;
+        font-size: 0.8em;
     }
 
     textarea {
         font-family: 'Open Sans', sans-serif;
         border: none;
         margin: 0;
+        margin-top: 5px;
         padding: 10px;
         width: 100%;
-        background-color: #f7f7f7;
+        background-color: $textarea-background;
         box-sizing: border-box;
+        border-radius: 3px;
+        color: #fff;
+
+        &::placeholder {
+            font-style: italic;
+        }
+    }
+
+    &__reply-input {
+        margin-top: 20px !important;
```

Comment:
Just checking the !important isn't just for testing?

Reply:
It's overriding the rule set in `.comment textarea`